docs/azure_arc_jumpstart/azure_arc_k8s/gke/gke_terraform/_index.md

```diff
@@ -61,174 +61,159 @@ The following Jumpstart scenario will guide you on how to use the provided [Terr
 
-* Export the environment variables needed for the Terraform plan.
+- Navigate to the [terraform folder](https://github.com/microsoft/azure_arc/tree/main/azure_arc_k8s_jumpstart/gke/terraform) and create a terraform.tfvars file with values for your environment.
 
   ```shell
-  export TF_VAR_subscriptionId='<Your Azure subscription ID>'
-  export TF_VAR_servicePrincipalAppId='<Your Azure service principal App ID>'
-  export TF_VAR_servicePrincipalSecret='<Your Azure service principal App Password>'
-  export TF_VAR_servicePrincipalTenantId='<Your Azure tenant ID'
-  export TF_VAR_gcp_project_id='<Your GCP Project ID>'
-  export TF_VAR_location='<Azure Region>'
-  export TF_VAR_resource_group_name='<Azure resource group name>'
-  export TF_VAR_gcp_credentials_filename='<Location on the Keys JSON file>'
-  export TF_VAR_gcp_region='<GCP Region to deploy resources>'
-  export TF_VAR_gke_cluster_name='<GKE cluster name>'
-  export TF_VAR_admin_username='<GKE control plane administrator username>'
-  export TF_VAR_admin_password='<GKE control plane administrator password>'
-  export TF_VAR_gke_cluster_node_count='<GKE cluster node count>'
-  export TF_VAR_gke_cluster_node_machine_type='<GKE cluster node machine type>'
+  subscriptionId="<Your Azure subscription ID>"
+  servicePrincipalAppId="<Your Azure service principal App ID>"
+  servicePrincipalSecret="<Your Azure service principal App Password>"
+  servicePrincipalTenantId="<Your Azure tenant ID"
+  gcp_project_id="<Your GCP Project ID>"
+  location="<Azure Region>"
+  resource_group_name="<Azure resource group name>"
+  gcp_credentials_filename="<Location on the Keys JSON file>"
+  gcp_region="<GCP Region to deploy resources>"
+  gke_cluster_name="<GKE cluster name>"
+  admin_username="<GKE control plane administrator username>"
+  admin_password="<GKE control plane administrator password>"
+  gke_cluster_node_count="<GKE cluster node count>"
+  gke_cluster_node_machine_type="<GKE cluster node machine type>"
   ```  
 
   For example:
 
   ```shell
-  export TF_VAR_subscriptionId='XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX'
-  export TF_VAR_servicePrincipalAppId='XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX'
-  export TF_VAR_servicePrincipalSecret='XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX'
-  export TF_VAR_servicePrincipalTenantId='XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX'
-  export TF_VAR_location='eastus'
-  export TF_VAR_resourceGroup='Arc-GKE-Demo'
-  export TF_VAR_gcp_project_id='azure-arc-demo-111111'
-  export TF_VAR_gcp_credentials_filename='account.json'
-  export TF_VAR_gcp_region='us-west1'
-  export TF_VAR_gke_cluster_name='arc-gke-demo'
-  export TF_VAR_admin_username='arcdemo'
-  export TF_VAR_admin_password='ArcDemo1234567!!'
-  export TF_VAR_gke_cluster_node_count=1
-  export TF_VAR_gke_cluster_node_machine_type='n1-standard-2'
+  subscriptionId="ea2faf71-b368-4143-b5a5-4f6bc3c6a606"
```

Comment:
We should instruct them to use a tfvars file instead of keying values into environment vars.